src/main/java/org/generationcp/breeding/manager/listmanager/util/DropHandlerMethods.java

```diff
@@ -508,7 +508,7 @@ public void buttonClick(final com.vaadin.ui.Button.ClickEvent event) {
 	}
 
 	public GermplasmListData getListDataByListIdAndLrecId(final Integer listId, final Integer lrecid, final GermplasmList germplasmList) {
-		return this.inventoryDataManager.getLotCountsForListEntries(listId, Lists.newArrayList(lrecid)).get(0);
+		return this.inventoryDataManager.getLotCountsForListEntries(Lists.newArrayList(lrecid)).get(0);
```

Comment:
it seems getListData**ByListIdAnd**LrecId has some unused parameters

Reply:
done